vendor/lancier-monitoring/sensorbox-codec.yaml

```diff
@@ -0,0 +1,87 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: sensor.js
+  # Examples (optional)
+
+# Downlink encoder encodes JSON object into a binary data downlink (optional)
+uplinkDecoder:

```

Comment:
this example also throw error and if you see lines are duplicated, please erase duplicated lines.